Casks/p/psst.rb

```diff
@@ -1,25 +1,31 @@
 cask "psst" do
-  version "0.1.0,20250525.150831"
-  sha256 :no_check
+  version "2025.06.09-f80f468"
+  sha256 "1ad6058a6814da00c2a420d101737ffa6f5eb07aec2002f0c9f9a501ebaea0f8"
```

Comment:
```suggestion
  sha256 :no_check
```

Because the `url` is unversioned we cannot use a checksum here.